tests/arc/tarc_import_inline_iterators.nim

```diff
@@ -0,0 +1,12 @@
+discard """
+  cmd: "nim c --gc:arc $file"
```

Comment:
Can we convert these to use a single entry `matrix`?

(can be in a follow-up PR)

Reply:
Absolutely. 
I have the intention of going over every remaining test, adding proper descriptions, tags and gathering what can / should be moved to the specs. This can be done in the same effort.

How would this specific entry be expressed as a matrix? Does this extend to every other use of "cmd:"?

Reply:
going from memory, but I believe this is it: `matrix: "--gc:arc"`

Yeah, most `cmd` can be converted to this, but not all.